src/main/java/game/Move.java

```diff
@@ -0,0 +1,8 @@
+package game;
+
+import java.awt.*;
+
+public class Move {
+    public Point startPosition;
```

Comment:
These need to be getters and setters rather than public properties. Here's a good SO comment describing all the reasons why: [https://stackoverflow.com/questions/1568091/why-use-getters-and-setters-accessors](https://stackoverflow.com/questions/1568091/why-use-getters-and-setters-accessors)